components/Buttons/src/MDCFlatButton.m

```diff
@@ -16,12 +16,23 @@
 
 #import "MDCFlatButton.h"
 
+#import "MaterialShadowElevations.h"
 #import "private/MDCButton+Subclassing.h"
 
 static NSString *const MDCFlatButtonHasOpaqueBackground = @"MDCFlatButtonHasOpaqueBackground";
 
 @implementation MDCFlatButton
 
++ (void)initialize {
```

Comment:
I think this is ok now.
We have now done an experiment with calling super setters and that seems to clobber appearance set values.